tasks/push-to-cdn/push-to-cdn.yaml

```diff
@@ -0,0 +1,140 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: push-to-cdn
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to push content to Red Hat's CDN
+  params:
+    - name: dataPath
+      type: string
+      description: >
+        Path to the JSON string of the merged data to use in the data workspace.
+    - name: binaries_dir
+      type: string
+      description: Path where the binaries are stored in the workspace
+    - name: subdirectory
+      description: Subdirectory inside the workspace to be used
+      type: string
+      default: ""
+  workspaces:
+    - name: data
+      description: Workspace to save the results to
+  results:
+    - name: srcDestPath
+      type: string
+      description: The relative path in the workspace to the stored source:destination paths
+  steps:
+    - name: create-exodus-conf-file
+    # TODO: Remove after testing
+    # image: quay.io/redhat-appstudio/release-service-utils:digest
+      image: quay.io/distribution_relengs/publish-to-dev-portal-test-image
+      script: |
+        #!/bin/bash
+        set -e
+
+        DATA_FILE="$(workspaces.data.path)/$(params.dataPath)"
+        if [ ! -f "${DATA_FILE}" ] ; then
+            echo "No data JSON was provided."
+            exit 1
+        fi
+
+        EXODUS_ENV=$(jq -r '.spec.data.exodus.env' $DATA_FILE)
```

Comment:
I am quite sure this will just be `.exodus.env` the data file doesn't have a spec and the data key isn't contained in it. See https://github.com/konflux-ci/release-service-catalog/blob/development/tasks/apply-mapping/tests/test-apply-mapping.yaml#L27 for an example one

Reply:
update, ty!